### PR TITLE
Add natss channel to the addressable resolver cluster role

### LINKED ISSUE #518: Add NATS Channel to the addressable-resolver-clusterrole
`addressable-resolver-clusterrole` is missing since it was added to eventing/contrib/nats after merging nats CRD here

### PR DESCRIPTION
**Pull request by montaro**
Fixes  #518 

**Release Note**

```release-note
Add NATS Channel to the addressable-resolver-clusterrole
```

### CODE REVIEW

File: natss/config/200-addressable-resolver-clusterrole.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: natss-addressable-resolver
+  labels:
+    eventing.knative.dev/release: devel
```
Comment (grantr) [inline natss/config/200-addressable-resolver-clusterrole.yaml]: This label should be `contrib.eventing.knative.dev/release`

Comment (montaro) [reply natss/config/200-addressable-resolver-clusterrole.yaml]: @grantr thanks for the review. should it be changed to `release` in all occurrences?

Comment (grantr) [reply natss/config/200-addressable-resolver-clusterrole.yaml]: :+1: All release labels in this repo should look like this:

```yaml
labels:
  contrib.eventing.knative.dev/release: devel
```

The only difference is the addition of `contrib.` to the beginning of the label key.

Comment (montaro) [reply natss/config/200-addressable-resolver-clusterrole.yaml]: @grantr Gotcha! Thanks! ✌️

**Pull request merged**